draft-documents/satisfying-real-world-use-cases.md

```diff
@@ -247,14 +247,13 @@ Everywhere Else, Peruvian economist Hernando de Soto highlights a fundamental
 difference between Western liberal democracies and other regions: such
 democracies have functioning systems that enable the abstract representation of
 value. For example, land is not simply owned territory--it is private property
-registered with the state and represented by land deeds; the representation
-having the force of law in disputes.
+registered with the state and represented by land deeds; This abstrct representation of the value of the land that is owned, the deed, is the instramental tool used to access capital. The west also has functioning state institutions and systems of law that imbue the deeds with cultural and legal power. They have what could be considered high levels of social trust - as described by Steven M.R. Covey in The Speed of Trust and summarized by one of the co-authors oof this document - Kaliya Identity Woman http://www.identitywoman.net/the-trouble-with-trust-the-case-for-accountability-frameworks
```

Comment:
When listing references, you can insert links with forms like `[text](URL)` in markdown; this is much easier to read. and [looks like this](http://foo.bar/).